sdlib/core/stdc/stdio.d

```diff
@@ -1,5 +1,17 @@
 module core.stdc.stdio;
 
+enum DbgPrintBufferSize = 4096;
+
+static char[DbgPrintBufferSize] _dbgPrintBuffer;
+
+void printDebug() {
+	dbgprint(_dbgPrintBuffer.ptr);
+}
+
+void dbgprint(char* str) {
+	write(2, str, strnlen(str, DbgPrintBufferSize));
```

Comment:
Avoid magic constants.

Reply:
Also, snprintf returns how many characters where printed, so there is no need to call strlen here.